src/logic/collectibles/sources/Gnosis.ts

```diff
@@ -1,41 +1,39 @@
 import { Collectibles, NFTAsset, NFTAssets, NFTTokens } from 'src/logic/collectibles/sources/collectibles.d'
 import { sameAddress } from 'src/logic/wallets/ethAddresses'
 import NFTIcon from 'src/routes/safe/components/Balances/assets/nft_icon.png'
-import { fetchErc20AndErc721AssetsList, fetchSafeCollectibles } from 'src/logic/tokens/api'
+import { fetchSafeCollectibles } from 'src/logic/tokens/api'
 import { TokenResult } from 'src/logic/tokens/api/fetchErc20AndErc721AssetsList'
 import { CollectibleResult } from 'src/logic/tokens/api/fetchSafeCollectibles'
-import { sameString } from 'src/utils/strings'
+import { TokenType } from 'src/logic/safe/store/models/types/gateway.d'
```

Comment:
@nicosampler I meant that you can import the existing enum here.

Reply:
mmm, I see.
Well syntactically they are the same, but semantically one represents the types from the gateway and the other an internal type from transactions.
Definitely, we should fix it but I'm not sure what is best, is it better to use the from `transactions.d` and delete the one from `gateway.d` or add it in `gateway.d` the same enum?

Reply:
Makes sense! Next time then 🙂

Reply:
> or add it in gateway.d the same enum?
Yeah, I guess it would be best to move it to the gateway in future.

Reply:
Sidenote: probably we shouldnt use .d.ts, they were not intended for it: https://stackoverflow.com/questions/21247278/about-d-ts-in-typescript

Reply:
@mikheevm so we should not write .d.ts at all, right? That is what I've interpreted from here: https://stackoverflow.com/a/54240522/2524608.

Regarding how to import the existing ones, we are doing it with only `.d` in the whole project and if you want to add `.d.ts` TS complains about it.

Reply:
They're intended for declaring already existing files. So if you have code in `file.js` and you need to declare it, you'd write `file.d.ts`

Reply:
exactly, all our code is in `.ts` files. So we are using it incorrectly. 

CC: @fernandomg @dasanra

Reply:
I guess since the folder is called `types`, it already tells us that the modules contain _only_ type definitions. So we can simply call them `file.ts`, w/o the `d`.

Reply:
can someone create a ticket with these suggested changes?